recipes/BEHST/build.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+PREFIX=.
+
+cd $PREFIX/bin/
+
+
+./download_behst_data.sh
```

Comment:
Sorry I should update the simple case to:

```bash
mkdir -p $PREFIX/bin
cp executable $PREFIX/bin
```

You might want to take a look at some other recipes to see how things are done.